index.js

```diff
@@ -13,8 +13,24 @@ app.use(morgan('dev'));
 
 app.use(routes);
 
+app.use((req, res, next) => {
+  const error = new Error('NOT FOUND');
+  error.status = 404;
+  next(error);
+});
+
+app.use((error, req, res, next) => {
+  res.status(error.status || 500);
+  res.json({
+    error: {
+      message: error.message,
+    },
+  });
+});
+
 const { PORT } = process.env;
 
+// eslint-disable-next-line no-console
 app.listen(PORT, () => console.log(`server started on port ${PORT}`));
```

Comment:
if you are done using the console.log development tool, you should go ahead and remove it since we do not leave development tools in production code

Reply:
thanx for point this out, I have removed it.